src/common/shared/shared.c

```diff
@@ -1597,3 +1597,24 @@ Info_SetValueForKey(char *s, char *key, char *value)
 
 	*s = 0;
 }
+
+unsigned int
+NextPow2(unsigned int i)
```

Comment:
There's a better method for this https://graphics.stanford.edu/~seander/bithacks.html#RoundUpPowerOf2

Reply:
What do you think about replace all places like https://github.com/yquake2/yquake2/blob/master/src/client/refresh/gl1/gl1_image.c#L677 with new func.
```c
for (scaled_width = 1; scaled_width < width; scaled_width <<= 1)
	{
	}
```

And in new code use `NextPow2(1024 + 1) == 2048` to receive same behavior.